<launch>
  <node pkg="player" name="player_node" type="player_node.py" output="screen"/>
  <node pkg="player" name="camera_node" type="camera_node.py" output="screen"/>
  <node pkg="player" name="tf_node" type="tf_node.py" output="screen"/>
  <node pkg="player" name="laser_node" type="laser_node.py" output="screen"/>
  <node pkg="player" name="move_node" type="move_node.py" output="screen"/>
  <node pkg="player" name="odom_node" type="odom_node.py" output="screen"/>
  <node pkg="player" name="match_node" type="match_node.py" output="screen"/>
  <node pkg="player" name="referee_node" type="referee_node.py" output="screen"/>
</launch>
